contributor_docs/es/contributor_guidelines.md

```diff
@@ -0,0 +1,272 @@
+
+Esta plantilla provee campos que debes llenar con la siguiente información relevante: 
+
+1.	**Incrementar Accesibilidad:**  Este es un campo requerido en el cual debes explicar cómo la adición de la mejora o cambio a la funcionalidad existente, ayudará a [incrementar accesibilidad]( https://github.com/processing/p5.js/blob/main/contributor_docs/access.md) para las personas que han sido históricamente marginadas en las disciplinas de las artes y las tecnologías. **Ninguna propuesta será aceptada sin esta declaración**. Sin embargo, puedes añadir "No estoy seguro" a este campo, y comenzar una conversación con la comunidad para entender como tu propuesta puede abordar el tema de accesibilidad en p5.js.
+
+2.	**¿Cuál es la subárea más adecuada de p5.js?:** Esto añadirá automáticamente las [etiquetas]( https://github.com/processing/p5.js/blob/main/contributor_docs/issue_labels.md) relevantes a tu <em>Issue</em>, lo que nos ayudará a identificar y responder adecuadamente.
+
+3.	**Detalles de mejora de funcionalidades:** Aquí debes describir tu propuesta para la mejora de funcionalidad(es). Una buena propuesta generalmente incluye un caso de uso específico: qué, cuándo, cómo y por qué esta funcionalidad es necesaria.
+
+Para que las propuestas de mejora de funcionalidad(es) sean aceptadas, estas deben ser aprobadas por al menos un [supervisor de área o responsable de mantenimiento]( https://github.com/processing/p5.js#stewards), antes de que se comience a trabajar en el <em>Pull Request</em>.
+
+**No debes registrar un <em>Pull Request</em> ni comenzar a trabajar en cambios de código sin antes haber registrado un <em>Issue</em> correspondiente, o antes de que dicho <em>Issue</em> haya sido aprobado para su implementación**. Esto se debe a que no hay garantía de que la propuesta será aceptada. Cualquier <em>Pull Request</em> que se registre antes de que se haya aprobado un <em>Issue</em> correspondiente será cerrado hasta que el <em>Issue</em> haya sido aprobado previamente.
+
+### ["Sugerir una funcionalidad nueva"](https://github.com/processing/p5.js/issues/new?assignees=\&labels=Feature+Request\&projects=\&template=feature-request.yml)
+
+Esta plantilla debe ser usada si deseas proponer la adición de una nueva funcionalidad a p5.js. Por ejemplo, imagina que quieres añadir soporte para dibujar elementos `<table>`, propios de HTML, con una nueva función que se llame `createTable`. Si tu propuesta se superpone con otras propuestas de mejora existentes, te sugerimos que evalúes y elijas la plantilla que consideres más apropiada para tu caso (ya sea funcionalidad existente o funcionalidad nueva)
+
+Esta plantilla provee campos que debes llenar con la siguiente información relevante: 
```

Comment:
las lineas 168 a 174 estan repetidas y falta este fragmento:
Accordingly, the template form fields are nearly identical to the field of the "Existing Feature Enhancement." As such please see the [previous section](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md#existing-feature-enchancement) for details about how to fill in each field.

puedes revisar por favor?

Reply:
@holomorfo Esta repetición de instrucciones fue intencional. Debajo de **sugerir una mejora a una funcionalidad existente** he descrito los siguientes títulos: 
1.	**Mejorar la accesibilidad:**
2.	**¿Cuál es la subárea más adecuada de p5.js?:**
3.	**Detalles de mejora de funcionalidades:**

Debajo de **Sugerir una funcionalidad nueva** he descrito los siguientes títulos, con una modificación pequeña en el último: 
1.	**Mejorar la accesibilidad:**
2.	**¿Cuál es la subárea más adecuada de p5.js?:**
3.	**Detalles de la nueva funcionalidad:**

Esto lo he hecho para que las personas no tengan que devolverse en la documentación, y leer algo que esta debajo de otro título, precisamente omitiendo el párrafo: “Accordingly, the template form fields are nearly identical to the field of the "Existing Feature Enhancement." As such please see the [previous section](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md#existing-feature-enchancement) for details about how to fill in each field.”

Sé que requiere más líneas, y repetición, pero da más claridad (en mi opinión). Cuéntame que prefieres.

Reply:
Me parece bien aumentar la accesibilidad si es intencional

Reply:
Corrección implementada